README.rst

```diff
@@ -1831,7 +1831,7 @@ Choose from the list of available rules:
 
   Unary operators should be placed adjacent to their operands.
 
-* **visibility_required** [@PSR2, @Symfony, @PhpCsFixer, @PHP71Migration]
+* **visibility_required** [@PSR2, @Symfony, @PhpCsFixer, @PHP71Migration, @PHP73Migration]
```

Comment:
how is it related to 7.3 ?

Reply:
`@PHP73Migration` extends `@PHP71Migration`

Reply:
ah, fair